File: API/src/main/java/com/MateuszLebioda/OneCinema/Model/roomPlan/ScreeningRoomPlanRowApiModel.java
package com.MateuszLebioda.OneCinema.Model.roomPlan;

import java.util.List;

public class ScreeningRoomPlanRowApiModel {
    private List<ScreeningRoomPlanSeatApiModel> seats;
    private int row;

    public List<ScreeningRoomPlanSeatApiModel> getSeats() {
        return seats;
    }

    public void setSeats(List<ScreeningRoomPlanSeatApiModel> seats) {
        this.seats = seats;
    }

    public int getRow() {
        return row;
    }

    public void setRow(int row) {
        this.row = row;
    }
}

File: UI/src/app/modules/booking/pages/booking-process/components/booking-preparation/components/screening-room/models/api/screening-room-plan-row-api.model.ts
import {ScreeningRoomPlanSeatApiModel} from './screening-room-plan-seat-api.model';

export class ScreeningRoomPlanRowApiModel {
  public row: number;
  public seats: ScreeningRoomPlanSeatApiModel[];

  constructor() {
    this.seats = [];
  }
}
